build-system/tasks/release/index.js

```diff
@@ -22,7 +22,7 @@ const fs = require('fs-extra');
 const klaw = require('klaw');
 const path = require('path');
 const tar = require('tar');
-const {cyan, green} = require('kleur/colors');
+const {cyan, green} = require('../common/colors');;
```

Comment:
```suggestion
const {cyan, green} = require('../../common/colors');
```

Reply:
Caught this myself before noticing your comment